apps/nccl/include/nccl.h

```diff
@@ -69,6 +69,15 @@ typedef struct ncclConfig_v21700 {
         NCCL_CONFIG_UNDEF_INT                             /* splitShare */     \
   }
 
+/* NCCL malloc and free function for all types of NCCL optimizations
+ * (e.g. user buffer registration). The actual allocated size might
+ * be larger than requested due to granularity requirement. */
+ncclResult_t ncclMemAlloc(void** ptr, size_t size);
+ncclResult_t pncclMemAlloc(void** ptr, size_t size);
+
+ncclResult_t ncclMemFree(void* ptr);
```

Comment:
Please copy the comments for `ncclMemFree` as well

Reply:
> Please copy the comments for `ncclMemFree` as well

Line 72 includes the free function. There is no separate comments for ncclMemFree.